Dockerfile.localmodules

```diff
@@ -0,0 +1,54 @@
+# Build the manager binary
```

Comment:
nit: Maybe we can just name it `Dockerfile.local` ?

Reply:
Renamed the file.